spec/support/helpers/flickr_helper.rb

```diff
@@ -0,0 +1,80 @@
+module FlickrHelper
+  def stub_flickr_requests
+    recent_response = { "photos": {
+      "page": 1,
+      "pages": 100,
+      "perpage": 10,
+      "total": "1000",
+      "photo": [
+          { "id": "22026223630",
+            "media": "photo",
+            "owner": "56956901@N05",
+            "secret": "ed93e74060",
+            "server": "600",
+            "farm": 1,
+            "title": "Boat ride on the bund #shanghai",
+            "ispublic": 1,
+            "isfriend": 0,
+            "isfamily": 0 },
+          { "id": "22224646391",
+            "media": "photo",
+            "owner": "75547855@N00",
+            "secret": "edf901d271",
+            "server": "5801",
+            "farm": 6,
+            "title": "DSC_0468",
+            "ispublic": 1,
+            "isfriend": 0,
+            "isfamily": 0 }
+      ]}, "stat": "ok" }.to_json
+
+    search_response = { "photos": {
+      "page": 1,
+      "pages": "34959",
+      "perpage": 10,
+      "total": "349585",
+      "photo": [
+            { "id": "22026530338",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.